nimble/controller/src/ble_ll_conn.c

```diff
@@ -3698,6 +3698,19 @@ ble_ll_conn_rx_data_pdu(struct os_mbuf *rxpdu, struct ble_mbuf_hdr *hdr)
     os_mbuf_free_chain(rxpdu);
 }
 
+static void
+ble_ll_conn_check_phy_update(struct ble_ll_conn_sm *connsm, uint8_t opcode, uint8_t *ctrdata)
+{
+    uint8_t tx_phy;
+
+    tx_phy = ctrdata[0];
+
+    if (opcode == BLE_LL_CTRL_PHY_UPDATE_IND) {
+        connsm->phy_tx_transition = tx_phy;
+        return;
+    }
```

Comment:
this function is a bit weird and not only because of this redundant `return`, but also it does not return anything so in case `opcode` does not match it won't touch `ctrdata` so not sure how caller should know about this

Reply:
This function end up here because previous versions of the fix used it in couple of places and I agree name is bad. Looks I forgot to clean this patch. I'm going to get rid of it